index.html

```diff
@@ -121,17 +116,17 @@
 <h2>Sample expandable menu</h2>
 
 <div>Change the appearance of the button:
-  <juicy-popover position="right top" handleSelector=".handle" expandableSelector=".expandable">
-    <div slot="handle">Background color ▶</div>
+  <juicy-popover position="right top">
+    <div id="bkdcolor" slot="handle">Background color ▶</div>
 
     <div slot="expandable">
-      <button onclick="document.querySelector('div.handle').style.background= 'crimson'">Red</button>
+      <button onclick="document.querySelector('#bkdcolor').style.background= 'crimson'">Red</button>
```

Comment:
Changed in https://github.com/Juicy/juicy-popover/pull/14/commits/e1123235601f22a6b9f2807582fef882840313a4 .